internal/cmd/config.go

```diff
@@ -328,29 +328,25 @@ type ClusterDisasterRecoveryConfig struct {
 
 	Charts struct {
 		Ark struct {
-			Chart   string
-			Version string
-			Values  struct {
-				Image struct {
+			Chart        string
+			Version      string
+			Values       map[string]interface{}
+			PluginImages struct {
+				Aws struct {
 					Repository string `chartConfig:"repository"`
 					Tag        string `chartConfig:"tag"`
 					PullPolicy string `chartConfig:"pullPolicy"`
-				} `chartConfig:"image"`
-				AwsPluginImage struct {
+				} `chartConfig:"aws"`
+				Azure struct {
 					Repository string `chartConfig:"repository"`
 					Tag        string `chartConfig:"tag"`
 					PullPolicy string `chartConfig:"pullPolicy"`
-				} `chartConfig:"awsPluginImage"`
-				AzurePluginImage struct {
+				} `chartConfig:"azure"`
+				Gcp struct {
 					Repository string `chartConfig:"repository"`
 					Tag        string `chartConfig:"tag"`
 					PullPolicy string `chartConfig:"pullPolicy"`
-				} `chartConfig:"azurePluginImage"`
-				GcpPluginImage struct {
-					Repository string `chartConfig:"repository"`
-					Tag        string `chartConfig:"repository"`
-					PullPolicy string `chartConfig:"repository"`
-				} `chartConfig:"gcpPluginImage"`
+				} `chartConfig:"gcp"`
```

Comment:
**Question**: have you checked the output/tested the `/api/v1/orgs/{orgId}/helm/cluster-charts` to give similar output before and after these configuration changes?

Reply:
Yes,  values for velero chart now returns `image` and `resources` since those are the user defined items which gets merge with other programmatically setup items in values.